src/test/java/Homework15.java

```diff
@@ -0,0 +1,45 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class Homework15 {
+    @Test
```

Comment:
you can type variable here will be more readable String url = "https://bbb.testpro.io/";